components/expression/expression-loader.js

```diff
@@ -28,8 +30,9 @@ genotet.utils.inherit(genotet.ExpressionLoader, genotet.ViewLoader);
  * @param {string} conditionRegex Regex for experiment condition selection.
  * @override
  */
-genotet.ExpressionLoader.prototype.load = function(matrixName, geneRegex,
-    conditionRegex) {
+genotet.ExpressionLoader.prototype.load = function(matrixName, geneRegex, conditionRegex) {
+  this.data.genes = geneRegex.split('|');
```

Comment:
geneRegex and conditionRegex can be any regex. It doesn't make sense to get genes by splitting. You will get list of gene names and condition names from ajax returns. See [wiki](https://github.com/ViDA-NYU/genotet/wiki/Server-Queries-Documentation).